deploy/markets.ts

```diff
@@ -102,11 +102,22 @@ const initializeMarkets: DeployFunction = async (hre) => {
       })
     }
 
-    const tTokenStrategy = await deploy({
-      hre,
-      contract: market.strategy.name,
-      indent: 4,
-    })
+    let tTokenStrategy: Contract
+
+    if (market.strategy.name == 'TTokenCompoundStrategy_2') {
+      tTokenStrategy = await deploy({
+        hre,
+        contract: market.strategy.name,
+        indent: 4,
+        args: [await (await getNamedSigner('deployer')).getAddress()],
```

Comment:
we will also need to set an approval amount on the underlying asset to allow the TToken contract to pull funds from the address we set here